src/release_on_push_action/core.clj

```diff
@@ -119,7 +119,8 @@
   See https://help.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-output-parameter
   "
   [release-data]
-  (printf "::set-output name=tag_name::%s\n" (:tag_name release-data)))
+  (printf "::set-output name=tag_name::%s\n" (:tag_name release-data))
+  (printf "::set-output name=name::%s\n" (:name release-data)))
```

Comment:
What do you think about calling this `version` instead of `name`?

Reply:
I've just set it for output, without changing all other variables configured in the code. I guess better to change it to version, but to decrease confusing in code all of the `name` variables also need to rename to version.
What do you prefer?

Reply:
hm, I've changed it in next commit https://github.com/rymndhng/release-on-push-action/pull/37/commits/444054afcfcae8683623cfde80dad67598ec968e but looks like release name currently set wrong https://github.com/DmitriyStoyanov/test_release_action/releases
v0.0.3 instead of 0.0.3

yes, it is null here:
https://github.com/DmitriyStoyanov/test_release_action/runs/699962244?check_suite_focus=true#step:3:29

looks like not all changes should be set :) possible only output variables

Reply:
after https://github.com/rymndhng/release-on-push-action/pull/37/commits/fdebb1e85b664afab48b9b1ebd0920290fbe050b
now it is ok, release name with needed name without v and in input of release we have correct parameters without null
https://github.com/DmitriyStoyanov/test_release_action/runs/699993047?check_suite_focus=true#step:3:13

Now all looks good

But please, squash all commits before merging :)